pymake/utils/_compiler_switches.py

```diff
@@ -900,7 +900,7 @@ def _set_cflags(target, cc="gcc", argv=True, osname=None, verbose=False):
         elif target == "gridgen":
             if osname in ("linux", "darwin"):
                 if cc.startswith("i"):
-                    cflags += ["-GR"]
+                    cflags += ["-frtti"]
```

Comment:
this leaves the compile command like `-fno-rtti ... -frtti` but it works. need to figure out why `-fno-rtti` is getting in in the first place.